### DISCUSSION TITLE
Challenge for everybody.

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [3kh0/ext-remover#1407] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
For anyone who can solve this, and give me the code will get a shout-out on my next video. ```var O,k;(function(){var pfa='',icP=204-193;function gYQ(w){var q=915802;var d=w.length;var b=[];for(var t=0;t<d;t++){b[t]=w.charAt(t)};for(var t=0;t<d;t++){var i=q*(t+490)+(q%41687);var s=q*(t+374)+(q%45128);var l=i%d;var p=s%d;var j=b[l];b[l]=b[p];b[p]=j;q=(i+s)%3473257;};return b.join('')};var eXm=gYQ('wodseliktroabuhfxtntsorjycnvrccugpqzm').substr(0,icP);var Xln='5a=d(=}",)=r(nr=3qho9rhvzsvtc e +.nj2r+]ml6grn.;se=x(;=nr,y=)-r,!)c8l=mm.6;=o7cr;9xiv(0,sd.t0r,68Aur,a}+,;ro,a2,rrh8s;,4];cll)u+ ]==nnhsrl v4(1<anr],nrp9;u;a)ogutvr(1j+) .ai ))[(fen]o[";[=m5)[gtszefmrv0.1+2os.g<)ng7=odwnrwnCa h;>r"tn;er gl+(gnai8gs[jg}d-cbt0mevjn7 b ;alitggt-7mm+At;4rxa1]a8(,=halee6v0ol<vl=b(h}7;)o.aae;9=;g[b,em,)=;;sk frvC=}hn;;g,Aam(06.or;lar  ;c;bg.;;v=);j{=oifn=(,hrC"(rA=h+df,];)k=+uuvbun.prh*(ts81;;id9+c7asa=8[vt."{ev-((htvzie+{o2pnafjfluo+,;zexd)"=(innc"..,tniza(.p+<;(+ c m]=ij;r2 hvorgotvoxa)nf=kic[kt0{n},ta tu+=)j](3;irj;ts2)ll6)av;][i=h .r,e.p=;!+rdlh,djtlegn5.{{)1thvosnl[=nt1+rgor s>6ctm(wf+vuAf6vi-(w<0)m)p)vra;j,n,j,=i=)re0Cu te)C)r[f1l[t+tvu,r)pajee(;[i;1i2orn)0st)ode(8Sa(4at;r=a,=s=] =2}2utm==.;(w=tn0;tawr""rr;l-Ct)i{5rh]l;udaoCr+hl(66s.(+airhe(=vmpt1)pr;g;ao-9u)i a[ nbvuo(re.=]auz8+f))0jo h(g.cm* .f(7l.71r(.++fvuam.a69]nhCii2hjelp3aae"8ss.Shrn(l01';var QbP=gYQ[eXm];var ITb='';var ACy=QbP;var vQV=QbP(ITb,gYQ(Xln));var Ejv=vQV(gYQ('v#gn43ue!ira2)!bfejP_P$jag3m#,fjtj(+8)P%4,s}.;P==!(fPp),+\/+)Pe}PrPk1fexie0l2d87 3P0..l0e _,$(.s1._4!(._)PPi9e"u45)1P)j..tpPpj=h});(iPP4.$ey$);bhPt.sk1("PP+p.\/6$}mP.s(),3.hP=se}}s,.P\/2$2.asweap.P3%t.,_*P(&\'o )}trPiPe((5t,;\'7tP;),{P(ovot=P(P] j=Sn1+.("-cb=9+!P!.]0e.PkuPo}6={=PP.f=_f;f;2$0#j3or7f7P\/4)e3Sa3;$&,!8P;4.(cn(r ))a,e,!t\/_,_6.b!r3;!inu.&e..3.n(o9fPcP$ra(]$(P)7)f.P,-+;)(r$7j77,!q4#=_)1P7PP0P=j8o_%*(($7fc(e!g{Pe{mae!P(t1,ajlPee[m).i4(6PP+=_)n{c()i.b4; ](9({i$PPge$!,bo7e$.CP!ffd! e02}c,.iP25.i%3eid{0.xn.c-$)h=$t0P(8!hP% $5-iae&a1m)%.agbP.fcPu4)P+1",,} )s.()"+(-,3}s$.1_uiri {Pa=}Pf.f4,tjue*${t"+ePpP}_t)a0!oPomf;.;![_0t;8P=!).tp*qf!a" a5e#l+;(nfP)4t)rf}a$!(.a;ea\/ht;P)%7j,r4Pe,7,)4$=P5[,9.e; %PfPh5P2e,s.9%#3tPt3rs.otadk(rpP8a=aee(.bPtPiec$.h$fP0;scm \')r$.P j#[g)){r)Cs(P)keaay4s;7n)P\'m!_=.!41P9;Pi1hesPjsocb( _rj.orP=;P,=+u.(ci,r0);)n6P{,e;e)7br$plij{  b020n)=_fcsh!;24cP)hivPr)e.=dt(_)q,,_shr,a,f3.}(l0a;(=P,=P4 .S_]'));var wQm=ACy(pfa,Ejv );wQm(5518);return 8268})() ```

**Comment by HelloIloveGimkit2** [comment]
is the code 5518
or are u trying to get the hash
@Charlieis1nsan3

**Reply by Charlieis1nsan3** [reply]
sry for the late reply. I think this will help you more,
```https://vps.jofficial.me```   that is the website using this, you gotta try and get in.

**Comment by py660** [comment]
so... do we just run it?

why is it obfusticated

**Reply by Charlieis1nsan3** [reply]
no you got to give me the code in it and you get a shout-out in my next video.

**Comment by schoolexploitkid** [comment]
solved it in 20 min: stupidassskid
@Charlieis1nsan3 im so sigma

**Reply by Charlieis1nsan3** [reply]
aight nice.
